package.json

```diff
@@ -18,12 +18,12 @@
     "jquery": "^3.5.0",
     "localforage": "^1.10.0",
     "moment": "^2.29.4",
-    "octokit": "^3.1.1",
+    "octokit": "^3.1.2",
     "prop-types": "^15.7.2",
     "react": "18.2.0",
     "react-dom": "18.1.0",
     "react-native-config": "^1.4.5",
-    "react-native-onyx": "1.0.100",
+    "react-native-onyx": "1.0.122",
```

Comment:
I had to bump the version because 1.0.100: `required: { node: '16.15.1', npm: '8.11.0' },`